pkgs/development/compilers/idris2/default.nix

```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
     patchShebangs --build tests
   '';
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  makeFlags = [ "PREFIX=$(out) OS=" ];
```

Comment:
This will probably break builds on cygwin. Since there's a chance it might build there, I would prefer leaving the OS setting alone or at least only overwrite it when darwin.

Reply:
In any case it should have been at least `"PREFIX=$(out)" "OS="`